oasislmf/model_execution/runner.py

```diff
@@ -68,7 +69,7 @@ def custom_get_getmodel_cmd(
                 max_process_id,
                 os.path.abspath("analysis_settings.json"),
                 "input")
-            if coverage_output != '' and not gul_alloc_rule:
+            if coverage_output != '' and gul_legacy_stream:
```

Comment:
Would it be more readable to switch the order?:

```suggestion
            if gul_legacy_stream and coverage_output != '':
```